server/repository/src/migrations/v1_07_00/changelog_add_name_link_id.rs

```diff
@@ -52,20 +53,33 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
 
         -- Adding changelog.name_link_id
         PRAGMA foreign_keys = OFF;
-        ALTER TABLE changelog
-        ADD COLUMN name_link_id TEXT REFERENCES name_link(id);
-        
-        UPDATE changelog
-        SET name_link_id = name_id;
+
+        CREATE TABLE tmp_changelog AS SELECT * FROM changelog;
+        DROP TABLE changelog;
+        CREATE TABLE changelog (
+            cursor INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+            -- the table name where the change happened
+            table_name TEXT NOT NULL,
+            -- row id of the modified row
+            record_id TEXT NOT NULL,
+            row_action TEXT NOT NULL,
+            -- Below fields are extracted from associated record where it's deemed necessary (see changelog/README.md)
+            name_link_id TEXT REFERENCES name_link(id), -- RENAMED from name_id
+            store_id TEXT,
+            is_sync_update BOOLEAN NOT NULL DEFAULT FALSE
+        );
+        INSERT INTO changelog SELECT * FROM tmp_changelog;
+        DROP TABLE tmp_changelog;
+
         PRAGMA foreign_keys = ON;
+        PRAGMA wal_checkpoint(TRUNCATE);      
```

Comment:
Nah it's not needed. Have removed.
Does feel like an SQLite bug, the drop column feature is relatively new and we're not on the latest sqlite possibly lots of fixed bugs since our version?

Definitely could be some other errors though...